### ISSUE TITLE
Cant install eos 1.8.1 using normal build

### ISSUE THREAD

**Issue by davidfrothin** [EOSIO/eos#7627]
I run the following steps for months now 

cd /home/ubuntu/eos
git pull --all
git checkout v1.8.1
git submodule sync
git submodule update --init --recursive
cd /home/ubuntu/eos/scripts
sudo ./eosio_uninstall.sh
./eosio_build.sh -s "EOS"
sudo ./eosio_install.sh

but upgrading to 1.8.1 causes errors ./eosio_build.sh -s "EOS" completes correctly but later running sudo ./eosio_install.sh gives the error /home/ubuntu/eos/scripts/eosio_install.sh
Please run /home/ubuntu/eos/scripts/eosio_build.sh first!

Please let me know what i am doing wrong and how to fix it.

**Comment by davidfrothin** [comment]
If i checkout as below then it is fine. Not sure what is wrong with 1.8.0 / 1.8.1

git checkout v1.8.0-rc2

please let me know what i need to change above

**Comment by NorseGaud** [comment]
The install script looks for a build/CMakeLists.txt file in the repo root. Are you moving the build directory outside of the repo root before running the install script?

**Comment by NorseGaud** [comment]
It could be an issue with running as sudo too.

**Comment by maoueh** [comment]
it's currently (https://github.com/EOSIO/eos/blob/master/scripts/eosio_install.sh#L43)

```
[ -f ${BUILD_DIR}/CMakeCache.txt ]] && printf "${COLOR_RED}Please run ${SCRIPT_DIR}/eosio_build.sh first!${COLOR_NC}" && exit 1
```

It should be this no?

```
[ ! -f ${BUILD_DIR}/CMakeCache.txt ]] && printf "${COLOR_RED}Please run ${SCRIPT_DIR}/eosio_build.sh first!${COLOR_NC}" && exit 1
```

**Comment by NorseGaud** [comment]
Yep, good catch. It must have been removed accidentally. We’ll get the fix in soon.

**Comment by NorseGaud** [comment]
In the meantime, please just cd into the build directory and run `make install`

**Comment by davidfrothin** [comment]
> In the meantime, please just cd into the build directory and run `make install`
that works thanks.

Do you recommend any changes to my script below after uninstalling previous nodeos version?

cd /home/ubuntu/eos
git pull --all
git checkout v1.8.1
git submodule sync
git submodule update --init --recursive
cd /home/ubuntu/eos/scripts
./eosio_build.sh -s "EOS"
./eosio_install.sh

**Comment by NorseGaud** [comment]
Sure, you might want to delete the /home/ubuntu/eos/build directory before running the build script. There can be weird things that happen if the CMakeCache.txt exists from an old build.

Also, you can ./scripts/eosio_build.sh instead of having to cd into the scripts path.

**Comment by ndcgundlach** [comment]
fixed with #7630
